int servo_id
double position
double velocity
double torque
